examples/NetworkAnalysis/Transmitter/Transmitter_esp8266

```diff
@@ -0,0 +1,76 @@
+// a 10K pull-down resistor is required
+// place it between the GND and the communications pin
+// if you are getting a report of every message in Busy bucket, add a resistor
+
+#if defined(ESP8266)
```

Comment:
I think it depends on the toolchain one compiles in, whether it's  included automatically or no. Conditional include, I guess, just defensive programming, so it still compiles if someone tries to put it on e.g. Uno.